Gemfile

```diff
@@ -49,9 +40,7 @@ group :development do
 end
 
 group :development, :test do
-  gem 'awesome_print'
   gem 'bullet'
-  gem 'bundler-audit', '>= 0.5.0', require: false
```

Comment:
No, you're thinking of `bummr`. `bundler-audit` is used to find CVEs and other Gemfile offenses. It's similar to Gemnasium, but it's already enabled in this repo's [Code Climate config](https://github.com/18F/identity-dashboard/blob/master/.codeclimate.yml#L4-L5), so unless devs need it to run locally, it's fine to remove.